src/kv/idbstore.rs

```diff
@@ -106,84 +112,37 @@ pub struct IdbStore {
 
 impl IdbStore {
     pub async fn new(name: &str) -> Result<IdbStore> {
-        let lc = LogContext::new();
-        let request = open_database(name)?;
-        let (callback, receiver) = IdbStore::oneshot_callback(lc.clone());
-        let request_copy = request.clone();
-        let closure_lc = lc.clone();
-        let onupgradeneeded = Closure::once(move |_event: web_sys::IdbVersionChangeEvent| {
-            let result = match request_copy.result() {
-                Ok(r) => r,
-                Err(e) => {
-                    error!(closure_lc, "Error before ugradeneeded: {:?}", e);
-                    return;
-                }
-            };
-
-            let db = web_sys::IdbDatabase::unchecked_from_js(result);
-
-            if let Err(e) = create_object_store(&db) {
-                error!(closure_lc, "Create object store failed: {:?}", e);
-            }
-        });
-        request.set_onsuccess(Some(callback.as_ref().unchecked_ref()));
-        request.set_onerror(Some(callback.as_ref().unchecked_ref()));
-        request.set_onupgradeneeded(Some(onupgradeneeded.as_ref().unchecked_ref()));
-        receiver.await?;
-
-        let db: IdbDatabase = request.result()?.into();
-        let db_copy = db.clone();
-        let onversionchange = Closure::once(move |_event: web_sys::IdbVersionChangeEvent| {
-            db_copy.close();
-        });
-        db.set_onversionchange(Some(onversionchange.as_ref().unchecked_ref()));
-        onversionchange.forget();
```

Comment:
I started all of this work because of this line. I was feeling anxiety about this and thought that it might have been the reason for the slowdown. It was not.

But after going down this route I felt that this PR was an approvement even if it does not fix the issue.